namespaces.go

```diff
@@ -255,21 +270,75 @@ func (nc nsCollector) describe(ch chan<- *prometheus.Desc) {
 	}
 }
 
+func (nc nsCollector) parseStorage(s string, d string) (string, error) {
```

Comment:
Could you add a comment with some input example? Makes it easier to follow what's going on.

Reply:
Thanks, added note in the function